AppImage x86 workflow randomly failing during libjxl build

Some workflow runs stall for a few minutes during or around the linking steps of the libjxl build/installation step before getting terminated.

Example:
```
2026-03-22T23:31:05.8330998Z [ 80%] Building CXX object lib/CMakeFiles/jpegli-static.dir/jpegli/render.cc.o
2026-03-22T23:31:05.8348941Z [ 81%] Building CXX object lib/CMakeFiles/jpegli-static.dir/jpegli/simd.cc.o
2026-03-22T23:31:05.8435002Z [ 81%] Building CXX object lib/CMakeFiles/jpegli-static.dir/jpegli/source_manager.cc.o
2026-03-22T23:31:05.8442453Z [ 81%] Building CXX object lib/CMakeFiles/jpegli-static.dir/jpegli/upsample.cc.o
2026-03-22T23:31:17.7805840Z [ 82%] Linking CXX static library libhwy_test.a
2026-03-22T23:36:02.7571331Z gmake[2]: *** Deleting file 'third_party/highway/libhwy_test.a'
2026-03-22T23:36:02.7588345Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_extras_jpegli-obj.dir/extras/enc/jpegli.cc.o'
2026-03-22T23:36:02.7596574Z gmake[2]: *** [lib/CMakeFiles/jxl_extras_jpegli-obj.dir/build.make:93: lib/CMakeFiles/jxl_extras_jpegli-obj.dir/extras/enc/jpegli.cc.o] Terminated
2026-03-22T23:36:02.7609413Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_cms.dir/jxl/cms/jxl_cms.cc.o'
2026-03-22T23:36:02.7653138Z gmake[2]: *** [lib/CMakeFiles/jxl_cms.dir/build.make:79: lib/CMakeFiles/jxl_cms.dir/jxl/cms/jxl_cms.cc.o] Terminated
2026-03-22T23:36:02.7668511Z gmake[1]: *** [CMakeFiles/Makefile2:1094: lib/CMakeFiles/jxl_extras_jpegli-obj.dir/all] Terminated
2026-03-22T23:36:02.8040811Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jpegli-static.dir/jpegli/upsample.cc.o'
2026-03-22T23:36:02.8056998Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jpegli-static.dir/jpegli/input.cc.o'
2026-03-22T23:36:02.8087525Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jpegli-static.dir/jpegli/entropy_coding.cc.o'
2026-03-22T23:36:02.8114846Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jpegli-static.dir/jpegli/color_transform.cc.o'
2026-03-22T23:36:02.8131347Z gmake[2]: *** [third_party/highway/CMakeFiles/hwy_test.dir/build.make:102: third_party/highway/libhwy_test.a] Terminated
2026-03-22T23:36:02.8151579Z gmake[2]: *** Deleting file 'tools/CMakeFiles/enc_fast_lossless.dir/__/lib/jxl/enc_fast_lossless.cc.o'
2026-03-22T23:36:02.8162380Z gmake: *** [Makefile:146: all] Terminated
2026-03-22T23:36:02.8181993Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/jpeg/dec_jpeg_data_writer.cc.o'
2026-03-22T23:36:02.8433967Z gmake[2]: *** [tools/CMakeFiles/enc_fast_lossless.dir/build.make:79: tools/CMakeFiles/enc_fast_lossless.dir/__/lib/jxl/enc_fast_lossless.cc.o] Terminated
2026-03-22T23:36:02.8463996Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/render_pipeline/stage_ycbcr.cc.o'
2026-03-22T23:36:02.8478719Z gmake[2]: *** [lib/CMakeFiles/jpegli-static.dir/build.make:79: lib/CMakeFiles/jpegli-static.dir/jpegli/adaptive_quantization.cc.o] Terminated
2026-03-22T23:36:02.8499636Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_extras_internal-obj.dir/extras/tone_mapping.cc.o'
2026-03-22T23:36:02.8547450Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_extras_internal-obj.dir/extras/metrics.cc.o'
2026-03-22T23:36:02.8601039Z gmake[2]: *** [lib/CMakeFiles/jxl_extras_internal-obj.dir/build.make:121: lib/CMakeFiles/jxl_extras_internal-obj.dir/extras/packed_image_convert.cc.o] Terminated
2026-03-22T23:36:02.8622724Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/render_pipeline/stage_upsampling.cc.o'
2026-03-22T23:36:02.8654284Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/render_pipeline/stage_tone_mapping.cc.o'
2026-03-22T23:36:02.8682868Z gmake[2]: *** Deleting file 'tools/CMakeFiles/jxl_tool.dir/tracking_memory_manager.cc.o'
2026-03-22T23:36:02.8709418Z gmake[2]: *** [tools/CMakeFiles/jxl_tool.dir/build.make:79: tools/CMakeFiles/jxl_tool.dir/cmdline.cc.o] Terminated
2026-03-22T23:36:02.8755618Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/render_pipeline/stage_gaborish.cc.o'
2026-03-22T23:36:02.8778492Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_dec-obj.dir/jxl/convolve_symmetric3.cc.o'
2026-03-22T23:36:02.8798778Z gmake[2]: *** [lib/CMakeFiles/jxl_dec-obj.dir/build.make:177: lib/CMakeFiles/jxl_dec-obj.dir/jxl/compressed_dc.cc.o] Terminated
2026-03-22T23:36:02.8813064Z gmake[2]: *** Deleting file 'lib/CMakeFiles/jxl_enc-obj.dir/jxl/enc_xyb.cc.o'
2026-03-22T23:36:02.8822402Z gmake[2]: *** [third_party/highway/CMakeFiles/hwy_list_targets.dir/build.make:79: third_party/highway/CMakeFiles/hwy_list_targets.dir/hwy/tests/list_targets.cc.o] Terminated
2026-03-22T23:36:02.8835530Z Terminated
2026-03-22T23:36:02.8852135Z gmake[2]: *** [lib/CMakeFiles/jxl_enc-obj.dir/build.make:79: lib/CMakeFiles/jxl_enc-obj.dir/jxl/butteraugli/butteraugli.cc.o] Terminated
2026-03-22T23:36:02.9178627Z ##[error]The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
2026-03-22T23:36:02.9186705Z ##[error]Process completed with exit code 143.
2026-03-22T23:36:03.0436641Z Cleaning up orphan processes
```

--- a/tools/makedeb/PKGBUILD.libjxl
+++ b/tools/makedeb/PKGBUILD.libjxl
@@ -34,7 +34,7 @@ build() {
   )
 
   cmake "${_cmake_options[@]}"
-  cmake --build build --parallel
+  cmake --build build --parallel $(nproc)
 }
 
 package() {

